tests/src/JIT/Performance/CodeQuality/Span/SpanBench.cs

```diff
@@ -412,13 +500,31 @@ public static void TestSpanDangerousCreateByte(int length)
         [InlineData(1000)]
         public static void TestSpanDangerousCreateString(int length)
         {
-            TestClass<string> testClass = new TestClass<string>();
-            testClass.C0 = new string[length];
-            Span<string> span;
-            foreach (var iteration in Benchmark.Iterations)
-                using (iteration.StartMeasurement())
-                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
-                        span = Span<string>.DangerousCreate(testClass, ref testClass.C0[0], testClass.C0.Length);
+            InvokeTestSpanDangerousCreate<string>(length);
+        }
+
+        static void InvokeTestSpanDangerousCreate<T>(int length)
+        {
+            TestClass<T> testClass = new TestClass<T>();
```

Comment:
Inside `Invoke`, we [start measuring](https://github.com/JosephTremoulet/coreclr/blob/198fa375dde167a433167fd4575846768879ffce/tests/src/JIT/Performance/CodeQuality/Span/SpanBench.cs#L73) just before invoking the delegate that's bound to the [lambda body](https://github.com/JosephTremoulet/coreclr/blob/198fa375dde167a433167fd4575846768879ffce/tests/src/JIT/Performance/CodeQuality/Span/SpanBench.cs#L511) that calls `TestSpanDangerousCreate<byte>`, and stop measuring [once that delegate call returns](https://github.com/JosephTremoulet/coreclr/blob/198fa375dde167a433167fd4575846768879ffce/tests/src/JIT/Performance/CodeQuality/Span/SpanBench.cs#L75).